extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/attribute/RequestLineAttribute.java

```diff
@@ -23,8 +23,20 @@ public String readAttribute(final RoutingContext exchange) {
                 .append(exchange.request().method())
                 .append(' ')
                 .append(exchange.request().uri());
-        sb.append(' ')
-                .append(exchange.request().version());
+        sb.append(' ');
+        String httpVersion = null;
```

Comment:
Should we have something like `"-"` as the default value? Because there's a good chance we will forget updating this code when they introduce 2.1 :).

Reply:
You are right, I think that's better than having the literal "null" printed out. I'll update this PR.

Reply:
Done. Updated the PR to use `-` as the default instead of `null`